MANIFEST.in

```
@@ -1,42 +1,16 @@
 include README.rst
 include CHANGES.rst
-include LICENSE.rst
-
-include ez_setup.py
-include ah_bootstrap.py
 include setup.cfg
-include specutils/tests/coveragerc
+include LICENSE.rst
```

I think this should be 
```suggestion
include licenses/LICENSE.rst
```
Or even just
```suggestion
```
since the `recursive-include licenses *` is already below